### ISSUE TITLE
[feature request] Configuration setting for download directory

### ISSUE THREAD

**Issue by xxxserxxx** [wangp/bower#83]
Attachments are currently saved in whichever directory bower was started from. It'd be nice to have a configuration option to set the default save-to directory.

**Comment by wangp** [comment]
Perhaps it can go in the `[ui]` section, e.g. `ui.default_save_directory`?

The first save command could use that for the suggested path, and subsequent save commands use the same directory as the previous save path (which is the current behaviour). If you want to write to the current directory, you would need to delete the directory part.

On the other hand, instead of adding a config option, you could just change the current directory before starting bower.

**Comment by xxxserxxx** [comment]
I do like the current "use previous directory" function, and yes, chdir to the downloads directory is a work-around; I could alias bower out to do this instead. While I believe this is a common configuration feature in most applications that support downloading, I will not be offended if you close this with "will not implement."
